### ISSUE TITLE
Question About Convert.py and CPU Offloading

### ISSUE THREAD

**Issue by yoponchik** [graphdeco-inria/gaussian-splatting#756]
When I run convert.py my CPU usage is over 70% and GPU is only 1%. Is that normal? 
Is there a way to make my GPU do the work?

**Comment by tapowanliwuyun** [comment]
From the code perspective, I believe the default should be to utilize the GPU for matching tasks. However, when I tried it on two different computers, the execution time varied significantly, with one taking only a few milliseconds and the other taking several hundred milliseconds. I suspect the former is running on the GPU while the latter is on the CPU, as I haven't made any other modifications, used the same instructions, and worked with the same dataset. I'm puzzled as to why this discrepancy occurred.

**Comment by tapowanliwuyun** [comment]
@yoponchik  Yes, the '-no_gpu' command is used to force the usage of the CPU, meaning that by default, the GPU is used for computation.

In fact, while computing poses using COLMAP on the computer where the matching block takes just a few milliseconds, the GPU usage significantly increases. I need to further determine the GPU usage on a computer where COLMAP takes a few hundred milliseconds to compute.
